### PR TITLE
Created ExcelToolkitWiki.cs

### LINKED ISSUE #30: ExcelToolkitWiki
#### Description:

Need a method to return valid wiki URL.

### PR DESCRIPTION
**Pull request by desaiwangBH**
### Issues addressed by this PR

Closes #30 

- method returns the wiki documentation link
- resolves reference to Excel_Toolkit wiki inside UI_Ribbon (see issue [here](https://github.com/BHoM/Excel_UI/issues/383)).

### CODE REVIEW

File: Excel_Engine/Query/ExcelToolkitWiki.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Generates the Excel Toolkit wiki URL for a provided page.")]
+        [Input("page", "An optional page in the wiki to link to. If no page is provided, the root URL is returned.")]
+        [Output("url", "Fully qualified URL for the Revit Toolkit wiki.")]
```
Comment (desaiwangBH) [inline Excel_Engine/Query/ExcelToolkitWiki.cs:40]: Excel Toolkit, not Revit!

Comment (FraserGreenroyd) [inline Excel_Engine/Query/ExcelToolkitWiki.cs:40]:
```suggestion
        [Output("url", "Fully qualified URL for the Excel Toolkit wiki.")]
```

**Review by FraserGreenroyd — APPROVED** [review]
Code changes look good

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core
@BHoMBot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning
@BHoMBot check null-handling
@BHoMBot check serialisation

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: versioning, ready-to-merge

**Pull request merged**